mathics/builtin/testing_expressions/list_oriented.py

```diff
@@ -39,6 +39,14 @@ class ArrayQ(Builtin):
      = False
     >> ArrayQ[{{a, b}, {c, d}}, 2, SymbolQ]
      = True
+    >> ArrayQ[SparseArray[{{1, 2} -> a, {2, 1} -> b}]]
+     = True
+    >> ArrayQ[SparseArray[{{1, 2} -> a, {2, 1} -> b}], 1]
+     = False
```

Comment:
Please keep in mind that this section is here to elucidate or educate users of some aspect of `ArrayQ`.  As such, usually we describe _why_ the example is given. For example:

```
'ArrayQ' works on sparse arrays:
>> ArrayQ[SparseArray[{{1, 2} -> a, {2, 1} -> b}]]
     = True
``` 

When one wants just wants to make sure all kinds of situations are covered, and this is not of that much interest to a user, then this goes should be added as part of a pytest.